ee_tests/src/specs/page_objects/app/header.ts

```diff
@@ -46,16 +46,19 @@ export class RecentItemsDropdown extends ui.Dropdown {
 
 
 export class Header extends ui.BaseElement {
+  mainNavBar: ui.BaseElement;
   profileDropdown = new ProfileDropdown(this.$('.pull-right.dropdown'));
   recentItemsDropdown = new RecentItemsDropdown(this.$$('.dropdown').get(0));
   // statusDropdown = new Dropdown(this.$$('.dropdown').get(0));
 
-  constructor(el: ElementFinder) {
-    super(el);
+  constructor(ele: ElementFinder = $('alm-app-header')) {
```

Comment:
Please pass in the object instead of setting the default as that makes it clear when you are initialising header what element it is pointing to.

Reply:
:+1: